keys/README.md

```diff
@@ -0,0 +1 @@
+This folder contains the identity files.
```

Comment:
You're right, we can let the user precise an absolute path, if the file doesn't exist, we look in ~/.ssh/ ?